app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,67 @@
+class AlbumsController < ApplicationController
+  def index
+    @albums = Album.order(ranking: :desc)
+  end
+
+  def show
+    id = params[:id]
+    @album = Album.find(id)
```

Comment:
It's better to replace these two lines with @album = Album.find(params[:id]), do the same in edit, update methods. You can then apply DRY principle: use before_action so u don't need to repeat the same line of code in all three places.